sdk/python/Makefile

```diff
@@ -5,7 +5,7 @@ SRC = pvsite_datamodel tests
 
 .PHONY: lint
 lint:
-	poetry run ruff check $(SRC)
+	poetry run ruff $(SRC)
 	# Sadly `ruff` does not cover everything, we still need black
 	poetry run black --check $(SRC)
```

Comment:
can this be set to 100 line length? Maybe it is already somewhere

Reply:
Yes it is in the `pyproject.toml`. Whenever possible I put all the settings in there.